README.md

```diff
@@ -215,3 +192,361 @@ if available(.settingsFeature) {
     )
 }
 ```
+
+## Code style
+
+The basis of the style of writing code is the configured Swiftlint in the project. The specific practices we follow are listed below.
+
+### Protocols
+Interfaces used for abstractions are named without prefixes and postfixes:
+
+🟢
+```
+protocol BuyService {}
+```
+
+🔴
+```
+protocol BuyServiceProtocol {}
+```
+
+
+For implementation or mocks we append `Impl`, `Mock` or other key words in the ending:
+
+🟢
+```
+final class BuyServiceImpl: BuyService {}
+final class BuyServiceMock: BuyService {}
+```
+
+🔴
+```
+final class ImplOfBuyService: BuyService {}
+final class BuyMockService: BuyService {}
+```
+
+
+If it is known in advance that only classes will conform to the protocol, you need to add the keyword AnyObject:
+
+🟢
+```
+protocol SomeProtocol: AnyObject {}
+```
+
+🔴
+```
+protocol SomeProtocol {}
+```
+
+
+### Class structure
+We try to avoid inheritance. For all classes from which inheritance is not planned, the final keyword must be explicitly specified. This speeds up the dispatching of calls in runtime and makes life easier for the compiler.:
+
+```
+final class BuyView: UIView {}
+```
+
+
+Dependencies and properties are always written at the top of the class, indicated by a comment without indentation after:
+
+🟢
+```
+final class SomeClass {
+
+    // Dependencies
```

Comment:
I suggest using "MARK: -", it might simplify searching for specific method/area in a code minimap. What do u think?

Reply:
I see it below, resolved